codeclimate-test-reporter.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |spec|
   spec.executables = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }.reject { |f| f == "ci" }
 
   spec.required_ruby_version = ">= 1.9"
-  spec.add_runtime_dependency "simplecov"
+  spec.add_runtime_dependency "simplecov", ">= 0.10", "<= 0.13"
```

Comment:
OK - I'm going to update to remove the minimum check, but keep the maximum check. I think that's the smallest change to address the current issue.